### PR TITLE
Remove moment.js; add bot response when new Thread is created

### PR DESCRIPTION
**Pull request by kien-ngo**

### CODE REVIEW

File: src/bot.js [hunk 1]
```diff
@@ -297,7 +297,7 @@ client.on("threadCreate", async (post) => {
   const forumTags = [];
 
   // Send message
-  const allowedChannels = []; // @Waren
+  const allowedChannels = []; // Only respond in appropriate channels
```
Comment (warengonzaga) [inline src/bot.js:300]: Hey @kien-ngo, kindly reference this back to the config file. Create a new entry there for this. The value should be in the config. This is for easy management. We don't need to change the code just to change the settings.

**Review by warengonzaga — CHANGES_REQUESTED** [review]
Kindy check the comment to the code.

File: src/bot.js (hunk 1 above)
Comment (kien-ngo) [reply src/bot.js:300]: @warengonzaga I'm going to remove this change so we can get this merged to "Dev"

Comment (warengonzaga) [reply src/bot.js:300]: I think I can handle it when it is available in the Dev branch.

**Comment by warengonzaga** [comment]
Hey @kien-ngo is this tested on your local?

**Comment by kien-ngo** [comment]
> Hey @kien-ngo is this tested on your local?

yeah. Can merge ^

**Review by warengonzaga — APPROVED** [review]
LGTM but will do some cleaning.

**Pull request merged**